crates/matrix-sdk/src/event_handler.rs

```diff
@@ -197,6 +198,7 @@ impl<E: std::error::Error> EventHandlerResult for Result<(), E> {
 }
 
 #[cfg(feature = "anyhow")]
+#[cfg(not(feature = "eyre"))]
```

Comment:
Is that `not` here needed? This means that you can't enable both features, or in other words if you enable both, both get disabled instead?